jobs/bump-jenkins-plugins.Jenkinsfile

```diff
@@ -0,0 +1,127 @@
+            }
+
+            stage("Check for plugin updates") {
+                def pluginsUpdate
+                pluginslist.each { plugin ->
+                    def parts = plugin.split(':')
+                    if (parts.size() == 2) {
+                        def pluginName = parts[0]
+                        def currentVersion = parts[1]
+                        pluginsUpdate = shwrapCapture("curl -Ls -o /dev/null -w '%{url_effective}' https://updates.jenkins.io/download/plugins/${pluginName}/latest/${pluginName}.hpi")
+                        def latestVersion = getPluginLatestVersion(pluginsUpdate)
+                        if (latestVersion.toString() != currentVersion.toString()) {
+                            haveChanges = true
+                            pluginsToUpdate["${pluginName}"] = [currentVersion, latestVersion]
+                            println("Plugin: ${pluginName} current version is ${currentVersion}, it will be updated to latest version: ${latestVersion}")
+                            shwrap("""
+                                cd fedora-coreos-pipeline
+                                sed -i 's/${pluginName}:${currentVersion}/${pluginName}:${latestVersion}/g' jenkins/controller/plugins.txt
+                            """)
+                        }
+                        else {
+                            println("The latest version of ${pluginName} is already installed: ${currentVersion}")
+                        }
+                    }
+                    else {
+                        println("ERROR: unexpected")
+                    }
+                }
+            }
+        
+            stage("Open a PR") {
+                if (haveChanges){
+                    def message = "bump jenkins plugin version"
+                    shwrap("""
+                        cd fedora-coreos-pipeline
+                        git add jenkins/controller/plugins.txt
+                        git commit -m '${message}' -m 'Job URL: ${env.BUILD_URL}' -m 'Job definition: https://github.com/coreos/fedora-coreos-pipeline/blob/main/jobs/bump-jenkins-plugins.Jenkinsfile'
+                    """)
+                    withCredentials([usernamePassword(credentialsId: botCreds,
+                                                      usernameVariable: 'GHUSER',
+                                                      passwordVariable: 'GHTOKEN')]) {
+                                                        shwrap("""
+                                                            cd fedora-coreos-pipeline
+                                                            git push -f https://\${GHUSER}:\${GHTOKEN}@github.com/${repo} ${pr_branch}
+                                                            curl -H "Authorization: token ${GHTOKEN}" -X POST -d '{ "title": "Bump jenkins plugin version to the latest", "head": "${pr_branch}", "base": "main" }' https://api.github.com/repos/coreosbot-releng/fedora-coreos-pipeline/pulls
```

Comment:
Can reuse `message` here.
We don't need string interpolation for `pr_branch`.
Should use `repo` variable.